tempesta_fw/sock_srv.c

```diff
@@ -414,6 +414,8 @@ tfw_sock_srv_disconnect(TfwConn *conn)
 	if (atomic_read(&conn->refcnt) != TFW_CONN_DEATHCNT)
 		ret = ss_close_sync(sk, true);
 
+	tfw_connection_release(conn);
```

Comment:
Normally connections are destroyed when a reference counter reach zero or `DEATH` value, and it's not obvious why need explicit `release()`  here. A comment is needed here or before function.